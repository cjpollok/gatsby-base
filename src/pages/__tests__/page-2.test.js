import React from 'react';
import { mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';

import SecondPage, { SecondPageBase } from '../page-2';

describe('SecondPage Component', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <MemoryRouter>
        <SecondPage />
      </MemoryRouter>
    );
    const base = wrapper.find(SecondPageBase);
    expect(base.render()).toMatchSnapshot();
  });
});
