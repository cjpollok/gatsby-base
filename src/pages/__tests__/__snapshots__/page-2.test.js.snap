// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondPage Component renders correctly 1`] = `
<a
  href="/"
>
  Go back to the homepage
</a>
`;
